<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>matchesUpdateForm</class>
 <widget class="QWidget" name="matchesUpdateForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="area" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, 
                                x1:0, y1:0, x2:0, y2:1, 
                                stop:0 rgba(6, 6, 6, 255), 
                                stop:0.41 rgba(56, 50, 52, 255), 
                                stop:1 rgba(34, 33, 33, 255));;
</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Azonix</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>Update Match :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_exit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  
border-radius:10px;
color:white;
}
QPushButton:hover {
    background-color:qlineargradient(x1: 0.25, y1: 0, x2: 0.75, y2: 1, 
                            stop: 0 rgba(0, 255, 254, 255),     
                            stop: 0.26 rgba(38, 170, 196, 255), 
                            stop: 0.67 rgba(89, 85, 131, 255),
                            stop: 1 rgba(0, 238, 255, 255));  
}</string>
    </property>
    <property name="text">
     <string>Save and Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  
border-radius:10px;
color:white;
}
QPushButton:hover {
    background-color:qlineargradient(x1: 0.25, y1: 0, x2: 0.75, y2: 1, 
                            stop: 0 rgba(0, 255, 254, 255),     
                            stop: 0.26 rgba(38, 170, 196, 255), 
                            stop: 0.67 rgba(89, 85, 131, 255),
                            stop: 1 rgba(0, 238, 255, 255));  
}</string>
    </property>
    <property name="text">
     <string>Cancel Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="matchIDLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Azonix</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:yellow;
</string>
    </property>
    <property name="text">
     <string>label</string>
    </property>
   </widget>
   <widget class="QComboBox" name="team1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
    border: 2px solid transparent; /* Default border */
}

/* Drop-down arrow area */
QComboBox::drop-down {
    border: none;
    width: 30px;
   ; /* Keep background for drop-down */
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
    image: url(:/textures/textures/downArrowIcon.png);
}

/* Hover effect */
QComboBox:hover {
    border: 3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));
}

/* Focus effect */
QComboBox:focus {
    border: 2px solid qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));
}

/* Style for the dropdown list */
QComboBox QAbstractItemView {
    background-color: white;
    border: 2px solid #C51D34;
    selection-background-color: #ff0004;
    selection-color: white;
    border-radius: 8px;
    color: black;
}
</string>
    </property>
   </widget>
   <widget class="QComboBox" name="team2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
    border: 2px solid transparent; /* Default border */
}

/* Drop-down arrow area */
QComboBox::drop-down {
    border: none;
    width: 30px;
   ; /* Keep background for drop-down */
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
    image: url(:/textures/textures/downArrowIcon.png);
}

/* Hover effect */
QComboBox:hover {
    border: 3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));
}

/* Focus effect */
QComboBox:focus {
    border: 2px solid qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));
}

/* Style for the dropdown list */
QComboBox QAbstractItemView {
    background-color: white;
    border: 2px solid #C51D34;
    selection-background-color: #ff0004;
    selection-color: white;
    border-radius: 8px;
    color: black;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="score">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
}

/* Hover effect */
QLineEdit:hover {
	border:3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}

/* Focus effect */
QLineEdit:focus {
    border:2px solid  qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}
</string>
    </property>
    <property name="placeholderText">
     <string>Updated Score</string>
    </property>
   </widget>
   <widget class="QComboBox" name="stadium">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
    border: 2px solid transparent; /* Default border */
}

/* Drop-down arrow area */
QComboBox::drop-down {
    border: none;
    width: 30px;
   ; /* Keep background for drop-down */
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
    image: url(:/textures/textures/downArrowIcon.png);
}

/* Hover effect */
QComboBox:hover {
    border: 3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));
}

/* Focus effect */
QComboBox:focus {
    border: 2px solid qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));
}

/* Style for the dropdown list */
QComboBox QAbstractItemView {
    background-color: white;
    border: 2px solid #C51D34;
    selection-background-color: #ff0004;
    selection-color: white;
    border-radius: 8px;
    color: black;
}
</string>
    </property>
    <property name="placeholderText">
     <string>Stadium</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>312</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDateEdit {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
    border: 2px solid transparent; /* Default border */
}

/* Drop-down arrow area */

/* Hover effect */
QDateEdit:hover {
    border: 3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));
}


</string>
    </property>
   </widget>
   <widget class="QLabel" name="mesupteam">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mesupteam2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mesupscore">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mesupstaduim">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mesupdate">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
